### DISCUSSION TITLE
Customizing the theme according to the documentation does not take effect ？

### DISCUSSION THREAD

**Discussion by FeiPengDev** [vueform/builder#77] [Questions] [answered]
I'm looking for a way to customize my theme, but after following the configuration in the documentation [themes](https://builder.vueform.com/docs/themes) and modifying some cssVars, it doesn't seem to work. 

![image](https://github.com/user-attachments/assets/16e9f5a0-8761-4e07-b026-10f393246c72)
![image](https://github.com/user-attachments/assets/953efe54-2391-4be8-82c5-9565885db47b)

**Comment by adamberecz** [comment, marked as answer]
Thank you for reporting the issue. A fix was released in `1.6.5`

**Comment by FeiPengDev** [comment]
fixed in 1.6.5
